essentials.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ath:
   useLocalSnapshots: false
-  athRevision: "1cc579eac831af27f444100c6bde73b342ec7c22"
+  athRevision: "1.78" # FTR 1.78 is 51a62122c5cda004db05599487abbb7069cf7b65
```

Comment:
use the actual tag name
```suggestion
  athRevision: "acceptance-test-harness-1.78" # FTR 1.78 is 51a62122c5cda004db05599487abbb7069cf7b65
```

Reply:
Oh FFS, :facepalm:. I tried to be smart, good demo I can not...
Apologies for the waste of time and thanks for the followup fix.